go/pserver/client.go

```diff
@@ -123,6 +123,10 @@ func (c *Client) FinishInitParams() error {
 // SendGrads sends gradients to parameter servers for updating
 // parameters.
 func (c *Client) SendGrads(grads []Gradient) error {
+	if len(grads) == 0 {
+		log.Info("Send Empty Gradient")
+		return nil
```

Comment:
Maybe return an error, we can treat it as error, so user can be aware of this error and fix it.